test/unit/package_test.rb

```diff
@@ -6,11 +6,11 @@
 module Packwerk
   class PackageTest < Minitest::Test
     setup do
-      @package = Package.new(name: "components/timeline", config: { "enforce_privacy" => "::Test" })
+      @package = Package.new(name: "components/timeline", config: { "enforce_privacy" => ["::Test"] })
```

Comment:
I'm pretty sure `application_validator.rb` would say that this is an invalid package:
https://github.com/Shopify/packwerk/blob/main/lib/packwerk/application_validator.rb#L101

So therefore I changed the test.